parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -210,18 +210,22 @@ write_files:
     {{if IsKataContainerRuntime }}
```

Comment:
I notice aks-e deprecated kata container runtime recently, shall we also remove it ?

https://github.com/Azure/aks-engine/pull/3014/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.t&file-filters%5B%5D=.yml&hide-deleted-files=true